UnityProject/Assets/Scripts/Explosions/ExplosionNode.cs

```diff
@@ -28,10 +32,10 @@ public void Process()
 			float EnergyExpended = 0;
 			var v3int = new Vector3Int(Location.x, Location.y, 0);
 
-			EnergyExpended = matrix.MetaTileMap.ApplyDamage(v3int, AngleAndIntensity.magnitude,
-				MatrixManager.LocalToWorldInt(v3int, matrix.MatrixInfo), AttackType.Bomb)*0.375f;
+			EnergyExpended = matrix.MetaTileMap.ApplyDamage(v3int, DD,
```

Comment:
might as well rename DD to something better

Reply:
did it

Reply:
cool